bson/src/test/unit/org/bson/DocumentTest.java

```diff
@@ -108,6 +111,53 @@ public void shouldGetWithDefaultValue() {
         assertEquals("bar", z2);
     }
 
+    @Test
+    public void shouldGetListFromDocument() {
+        // given
+        Document d = new Document("x", 1)
+                .append("y", Arrays.asList("two", "three"))
+                .append("z", Arrays.asList(new Document("a", 1), new Document("b", 2)))
+                .append("w", new Document("a", Arrays.asList("One", "Two")))
```

Comment:
`asList` has been statically imported so no need for the longer form `Arrays.asList`.